src/core/index.js

```diff
@@ -251,14 +251,19 @@ function extractAuthOptions(options) {
     params,
     autoParseHash,
     redirect,
+    redirectUri, //invalid prop. it's only here to show a warning
     redirectUrl,
     responseMode,
     responseType,
     sso,
     state,
     nonce
-  } =
-    options.auth || {};
+  } = options.auth || {};
+  if (redirectUri) {
```

Comment:
I'd keep the previous approach that didn't require you to expose a new variable to this function's scope. (rollback that last commit). Maybe keep the comment inside this IF clause @luisrudge

Reply:
done